src/status_im/commands/handlers/loading.cljs

```diff
@@ -95,18 +99,36 @@
 
 (defn add-commands
   [db [id _ {:keys [commands responses autorun]}]]
-  (let [account    @(subscribe [:get-current-account])
-        commands'  (filter-forbidden-names account id commands)
-        responses' (filter-forbidden-names account id responses)]
-    (-> db
-        (assoc-in [id :commands] (mark-as :command commands'))
-        (assoc-in [id :responses] (mark-as :response responses'))
-        (assoc-in [id :commands-loaded] true)
-        (assoc-in [id :autorun] autorun))))
+  (let [account        @(subscribe [:get-current-account])
+        commands'      (filter-forbidden-names account id commands)
+        global-command (:global commands')
+        commands''     (apply dissoc commands' [:init :global])
+        responses'     (filter-forbidden-names account id responses)]
+    (cond-> db
+
+            (get-in db [:chats id])
+            (update-in [:chats id] assoc
+                       :commands (mark-as :command commands'')
+                       :responses (mark-as :response responses')
+                       :commands-loaded true
+                       :autorun autorun
+                       :global-command global-command)
+
+            global-command
+            (update :global-commands assoc (keyword id)
+                    (assoc global-command :bot id
+                                          :type :command)))))
 
 (defn save-commands-js!
   [_ [id file]]
-  (commands/save {:chat-id id :file file}))
+  #_(commands/save {:chat-id id :file file}))
```

Comment:
1. We never use this data
2. I've chanaged `command` schema and meaning
3. We have to revisit this later and maybe just save command.js within `contact` entity